ParkJInjoo/app.js

```diff
@@ -0,0 +1,141 @@
+const http = require("http");
+
+require("dotenv").config();
+const express = require("express");
+const cors = require("cors");
+const morgan = require("morgan");
+const { DataSource } = require("typeorm");
+
+app = express();
+
+app.use(express.json());
+app.use(cors());
+app.use(morgan("combined"));
+
+const database = new DataSource({
+  type: process.env.TYPEORM_CONNECTION,
+  host: process.env.TYPEORM_HOST,
+  port: process.env.TYPEORM_PORT,
+  username: process.env.TYPEORM_USERNAME,
+  password: process.env.TYPEORM_PASSWORD,
+  database: process.env.TYPEORM_DATABASE,
+});
+
+database.initialize().then(() => {
+  console.log("Data Source has been initialized!");
+});
+
+app.post("/users", async (req, res) => {
+  const { name, email, profile_image, password } = req.body;
```

Comment:
실제 컬럼은 profile_image라고 명시되어 있지만, JavaScript 언어의 네이밍 컨벤션은 보통 Camel case로 작성합니다.

데이터베이스의 컬럼명이 아닌, 클라이언트로부터 요청 데이터를 받아오는 부분들 모두 Camel case로 수정해주세요!

```suggestion
  const { name, email, profileImage, password } = req.body;
```